src/main/java/org/candlepin/subscriptions/inventory/db/model/InventoryHost.java

```diff
@@ -67,13 +68,21 @@
         "h.facts->'rhsm'->>'orgId' as org_id, " +
         "h.facts->'rhsm'->>'CPU_CORES' as cores, " +
         "h.facts->'rhsm'->>'CPU_SOCKETS' as sockets, " +
-        "h.facts->'qpc'->>'IS_RHEL' as is_rhel, " +
         "h.facts->'rhsm'->>'SYNC_TIMESTAMP' as sync_timestamp, " +
-        "cj.products " +
+        "cj.products, " +
+        "qpc_prods.products as qpc_products, " +
+        "qpc_certs.product_ids as qpc_product_ids " +
         "from hosts h " +
         "cross join lateral ( " +
-            "select string_agg(cj.elem::text, ',') as products " +
-            "from json_array_elements_text(h.facts::json->'rhsm'->'RH_PROD') as cj(elem)) cj " +
+            "select string_agg(cj.elem, ',') as products " +
+            "from jsonb_array_elements_text(h.facts->'rhsm'->'RH_PROD') as cj(elem)) cj " +
+        "cross join lateral ( " +
+            "select string_agg(cj.elem, ',') as products " +
+            "from jsonb_array_elements_text(h.facts->'qpc'->'rh_products_installed') as cj(elem)) " +
```

Comment:
Is this right?  The clause is referenced as `qpc_prods` but inside the clause, you're using `cj`.  Shouldn't it be

```
cross join lateral (
  select string_agg(qpc_prods.elem, ',') as products
  from jsonb_array_elements_text(h.facts->'qpc'->'rh_products_installed') as qpc_prods(elem)) qpc_prods
```